app/src/main/java/com/infomaniak/mail/ui/MainActivity.kt

```diff
@@ -95,6 +95,7 @@ class MainActivity : ThemedActivity() {
     override fun onStart() {
         super.onStart()
         SyncMessagesWorker.cancelWork(this)
+        mainViewModel.cancelCurrentMailboxNotifications()
```

Comment:
This probably won't trigger at boot, because the Mailbox flow will not be set yet.

Reply:
Fixed.